onchain/rollups/contracts/dapp/CartesiDApp.sol

```diff
@@ -134,15 +136,11 @@ contract CartesiDApp is
         }
 
         // execute voucher
```

Comment:
Should we add an comment here annotating that it will revert if the `functionCall` fails? As the syntax does not make clear that its an Open Zeppelin import and the behavior is different from the usual call.

Reply:
This can be easily deducted from the fact that `functionCall` is not a built-in [member of addresses](https://docs.soliditylang.org/en/latest/types.html#members-of-addresses), and that we are using the `Address` library from OpenZeppelin for `address` values. [(1)](https://github.com/cartesi/rollups-contracts/blob/515477f6fff67c308153e5581388910cb84d40cf/onchain/rollups/contracts/dapp/CartesiDApp.sol#L15) [(2)](https://github.com/cartesi/rollups-contracts/blob/515477f6fff67c308153e5581388910cb84d40cf/onchain/rollups/contracts/dapp/CartesiDApp.sol#L68)

Also, I think it is unnecessary to say that a call to another contract might revert, and that any error will be propagated, beacuse this is the default behavior of calls in Solidity. Errors must be handled explicitly, either by low-level calls or through try-catch clauses.